### PR TITLE
Move rate limiter from client to jaeger-lib

### PR DESCRIPTION
**Pull request by isaachier**
Signed-off-by: Isaac Hier <ihier@uber.com>

### CODE REVIEW

File: utils/rate_limiter.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```
Comment (black-adder) [inline utils/rate_limiter.go]: although factually these files are just being copied over, shouldn't the copyright be updated here to 2018?

Comment (isaachier) [reply utils/rate_limiter.go]: Right I am wondering too. @yurishkuro what is your opinion here?

Comment (isaachier) [reply utils/rate_limiter.go]: He wrote to me yesterday: "if you copy the file, you keep C as is... if you copy and modify, and C was 2016, you should change it to C 2016, 2018"

**Comment by yurishkuro** [comment]
why do we need to move it? It's already a public class, can just use it directly from the go-client path.

**Comment by isaachier** [comment]
You mean the client is accessible from the main jaeger repo or do you mean the client has access to this jaeger-lib repo? I assumed not the former, only the latter.

**Comment by yurishkuro** [comment]
nvm, I am find to move it.

**Comment by yurishkuro** [comment]
:shipit:

**Comment by isaachier** [comment]
I'm still seeing I am not authorized to merge this.

File: utils/rate_limiter.go (hunk 1 above)
Comment (black-adder) [reply utils/rate_limiter.go]: update this and I'll land

Comment (yurishkuro) [reply utils/rate_limiter.go]: update to what? This is the correct header.

Comment (isaachier) [reply utils/rate_limiter.go]: See my response above from @yurishkuro (granted it is hearsay as he didn't write it here)? It was not modified, so keeping the same.

**Pull request merged**